polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -91,6 +91,28 @@ public static void enforceFeatureEnabledOrThrow(
           .defaultValue(false)
           .buildFeatureConfiguration();
 
+  /**
+   * When enabled, includes session tags (catalog, namespace, table, principal, request-id) in AWS
```

Comment:
nit: the javadoc is mostly equivalent to the flag's descriptions... why bother duplicating that info?

Reply:
Done. Removed the redundant javadoc comment since the description in the builder already documents the flag's behavior.